inc/change.class.php

```diff
@@ -1268,10 +1265,9 @@ function showPlanForm() {
 
 
    /**
+    * {@inheritDoc}
```

Comment:
see doc about that; should not be used

Reply:
https://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#override-method-inheritdoc-see-docblock-no-docblock